infrastructure/vars.tf

```diff
@@ -381,3 +381,15 @@ variable "db_accessor_instance_class" {
   type        = string
   default     = "t3.micro"
 }
+
+variable "elk_instance_class" {
+  description = "elk_instance_class"
+  type        = string
+  default     = "t3.micro"
+}
+
+variable "create_elk_instance" {
+  description = "Whether to create a DB accessor instance. This instance can be used to access RDS and is spun up in a private subnet. It can be accessed using AWS Systems Manager Session Manager."
```

Comment:
Should change the description here to reference the ELK instance instead of "DB accessor"